components/tidb_query_vec_expr/src/impl_compare.rs

```diff
@@ -227,6 +227,38 @@ pub fn coalesce<T: Evaluable>(args: &[&Option<T>]) -> Result<Option<T>> {
     Ok(None)
 }
 
+#[rpn_fn(varg)]
```

Comment:
I think it's vital to validate if the argument count is at least 2

Reply:
You're right about the arguments count validation, but why at least 2? I think one argument should be ok for this func.

BTW, to validate the argument count, is `#[rpn_fn(varg, min_args = 1)]` right to do this?

Reply:
Because `GREATEST()` in MySQL just requires there are at least two arguments...

Reply:
Yes. The way you validate it is right.

Reply:
Thanks for point out my mistake. In this case, should I remove the single argument unit test cases also?

Reply:
Ah, yes. You'd better remove it as it's an invalid case.